macx/scripts/osxdist.py

```diff
@@ -296,7 +296,8 @@ def package_client():
 	a.copy_resources(['icons/mumble.icns'])
 	a.update_plist()
 	if not options.universal:
-		a.add_compat_warning()
+		if options.compat_warning:
+			a.add_compat_warning()
```

Comment:
From the commit message:
> QMake always passes "-arch x86_64" after the parameters specified in our project file, meaning that we can't build the application as 32 bit.
>
> This option allows us to disable the feature (warning when trying to run Mumble on a 32 bit system) for the CI build.

Reply:
Any idea why it suddenly would do that and not before?

Reply:
We never built the installer on Travis CI before.

Reply:
Why would it matter whether it's in travis though?

Reply:
It shouldn't, indeed.

It's possible that `-arch -x86_64` is passed automatically in recent QMake versions.

Reply:
Does it work in internal CI? Do we have a way to experiment with that? My assumption is that adding such a binary is important for the osx bundle experience or @mkrautz wouldn't have bothered to add it in the first place.

Reply:
Yes, it works in internal CI, `-arch x86_64` is not passed there.

The binary's purpose is to show an informative message in case the bundle is run on a 32 bit system:
https://github.com/mumble-voip/mumble/blob/15072a4505af39f254fe0d35357d95b9a121e991/macx/compat/compat.m#L21-L40